exercises/sieve/example.rs

```diff
@@ -0,0 +1,33 @@
+pub struct Sieve { limit: usize }
+
+impl Sieve {
+    pub fn new(limit: usize) -> Sieve {
+        Sieve { limit: limit }
+    }
+
+    pub fn primes(&self) -> Vec<usize> {
+        let mut primes = (0..self.limit).map ( |x| x + 1).collect::<Vec<usize>>();
```

Comment:
Were I very concerned about performance I would wonder whether this could be expressed as (0..(self.limit+1)).skip(1) , though I am not so concerned here.